lib/jsdom/browser/new-resource-loader.js

```diff
@@ -0,0 +1,131 @@
+"use strict";
```

Comment:
This file is really nice and clear and readable; great work.

Reply:
Thanks! :)